### ISSUE TITLE
How to disable the log of boringproxy run as server?

### ISSUE THREAD

**Issue by coder-free** [boringproxy/boringproxy#128]
Too many log like this:
```
2022-01-04T23:59:42+08:00 127.0.0.1 GET boring.domain.com /api/tunnels
2022-01-04T23:59:44+08:00 [ GET boring.domain.com /api/tunnels
```

**Comment by anderspitman** [comment]
I agree! It's quite annoying. This is due to the polling nature of the clients. I'm leaving it in for now to remind me that I need to create a better synchronization method. I'll probably add an option for long-polling.

If I don't get to this soon, I'll just remove the logging.

**Comment by almereyda** [comment]
Slightly OT: The logging can also lead to interesting insights about the environment one runs the service within. https://forum.indiebits.io/t/trivia-identifying-automated-access-attempts/40
